Modules/Wiki/classes/class.ilObjWikiGUI.php

```diff
@@ -1631,7 +1629,7 @@ public function exportHTML() : void
      * Get title for wiki page (used in ilNotesGUI)
      */
     public static function lookupSubObjectTitle(
-        string $a_wiki_id,
+        string $a_wiki_id,// TODO PHP8-REVIEW This is a `string`, but it is passed to a constructor expecting and `int`. Furthermore it is compared to an `int` below.
```

Comment:
This is a `string`, but it is passed to a constructor expecting and `int`. Furthermore it is compared to an `int` below.